src/custom/components/CowSubsidyModal/constants.ts

```diff
@@ -1,10 +1,9 @@
-export const COW_SUBSIDY = [
-  ['(v)COW balance', 'Fee discount'],
+// used in modal and popover
+export const SUBSIDY_INFO_MESSAGE = 'As a (v)COW token holder you will be eligbible for a fee discount.'
+export const COW_SUBSIDY_DATA = [
   ['0', 0],
   ['100', 10],
   ['1000', 15],
   ['10000', 25],
   ['100000', 35],
```

Comment:
Just occurred to me.
When you compare  these values against the balance, is it comparing units or atoms?
It might as well be that these are atoms and everything will be 0%?

Reply:
just saw this comment but i switched it to atoms indeed, is a better check

Reply:
accidentally pushed these requested changes to the waterfall sequential PR #2558 

here is this change of atoms and some other test logic: [3df5f18](https://github.com/gnosis/cowswap/pull/2558/commits/3df5f18614c828cf47ac31ca2c2ad1992447f90b)